src/server.js

```diff
@@ -42,6 +42,20 @@ const renderImage = (params, response, next, tilePath) => {
                 )
             )
         }
+    } else if (typeof default_styles === 'string') {
+        try {
+            // read styleJSON
+            const data = fs.readFileSync(default_styles, "utf8")
```

Comment:
Is this correct?  Your usage below for the command line interface suggests a *directory* of default style files, whereas here we are treating it as a path to a specific style JSON file.

Reply:
I updated usage now. Path to specific style JSON is needed.